llvm/include/llvm/IR/DataLayout.h

```diff
@@ -343,13 +345,17 @@ class DataLayout {
 
   /// Return the address spaces containing non-integral pointers.  Pointers in
   /// this address space don't have a well-defined bitwise representation.
-  ArrayRef<unsigned> getNonIntegralAddressSpaces() const {
-    return NonIntegralAddressSpaces;
+  SmallVector<unsigned, 8> getNonIntegralAddressSpaces() const {
```

Comment:
Is this actually used for anything now?

Reply:
It is only used in tests - I could drop those if you prefer?

Reply:
No, it might be useful somewhere I suppose